Cargo.toml

```diff
@@ -9,12 +8,19 @@ members = [
     "simulation",
     "test-macros",
     "utils",
+    "workspace-hack",
     "xline",
     "xline-client",
     "xline-test-utils",
     "xlineapi",
     "xlinectl",
 ]
+resolver = "2"
+
+# TODO: when cargo-machete support ignore workspace member crate, then add it into pre commit hook & CI
+# ISSUE: https://github.com/bnjbvr/cargo-machete/issues/94
+[workspace.metadata.cargo-machete]
+ignored = ["prost", "workspace-hack"]
```

Comment:
Why do you ignore the `prost`?

Reply:
`prost` seems to be used in generated `xxxpb.rs` files in `target/`. It will not be searched by cargo machete.